rosidl_generator_c/resource/msg__functions.c.em

```diff
@@ -98,11 +101,12 @@ for field in spec.fields:
             elif field.default_value is not None:
                 # set default value of primitive type
                 lines.append('msg->%s = %s;' % (field.name, value_to_c(field.type, field.default_value)))
-
```

Comment:
As mentioned before please avoid whitespace changes like this and restore the empty line.